src/Application.php

```diff
@@ -170,8 +169,17 @@ public function writeFull(SplFileInfo $csv, array $tableConfig): void
 
     protected function reorderColumns(SplFileInfo $csv, array $items): array
     {
-        $csvReader = new CsvReader($csv->getPathname());
-        $csvHeader = $csvReader->getHeader();
+        $manifestPath = $csv->getPathname() . '.manifest';
```

Comment:
Riesim `db-writer-synapse`, ... potrebujem tam `storage-api-client` ... a ten pouziva `keboola/csv ~ 1.0` ... `db-writer-common` zase  `keboola/csv ~ 2.0`. Pozrel som sa na to podrobnejsie, a jedine miesto kde sa v `src` pouziva `keboola/csv` je metoda `reorderColumns`, ... no zoznam stlpcov vieme vytiahnut z manifestu, a nepotrebujeme tak citat CSVcko.